datadistribution.sh

```diff
@@ -13,20 +13,19 @@ requires:
   - protobuf
   - O2
   - fmt
+  - "ucx:(slc8)"
```

Comment:
Will this include the lib for both EPN and FLP builder?

Reply:
Unless the FLP default explicitly disables it, yes.